commercetools-models/src/test/java/io/sphere/sdk/carts/CartOriginIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.test.IntegrationTest;
+import io.sphere.sdk.test.SphereTestUtils;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static io.sphere.sdk.carts.CartFixtures.*;
+
+public class CartOriginIntegrationTest extends IntegrationTest {
```

Comment:
It really makes a lot sense to split our integration tests 🥇 May be we should think about a naming scheme for them in the future 😄

Reply:
Yep the naming doesen't have a clear convention now 👍